test/stringio/test_stringio.rb

```diff
@@ -483,6 +483,11 @@ def test_seek
     f.close unless f.closed?
   end
 
+  def test_seek_frozen_string
+    f = StringIO.new(-"1234")
+    assert_nothing_raised { f.seek(0) }
```

Comment:
In general, we should not use `assert_nothing_raised` because it doesn't nothing. We should check the read behavior instead. How about the following?

```suggestion
    f.seek(0)
    assert_equal("234", f.read)
```